python/tests/engine/test_python.py

```diff
@@ -1567,6 +1572,346 @@ def test_infer_type_pyarrow_struct_with_list_of_struct_fields(self):
             == "struct<user0:array<struct<value:array<double>>>,user1:array<struct<value:array<double>>>>"
         )
 
+    def test_infer_type_pandas_pyarrow_int8(self):
+        # Arrange
+        pdf = pd.DataFrame({"data": pd.Series([1, 5], dtype="int8[pyarrow]")})
+        arrow_schema = pa.Schema.from_pandas(pdf)
+
+        python_engine = python.Engine()
+
+        # Act
+        arrow_type = python_engine._convert_simple_pandas_dtype_to_offline_type(
+            arrow_schema.field("data").type
+        )
+        print(arrow_type)
+
+        # Assert
+        assert arrow_type == "int"
+
+    def test_infer_type_pandas_pyarrow_int16(self):
+        # Arrange
+        pdf = pd.DataFrame({"data": pd.Series([1, 5], dtype="int16[pyarrow]")})
+        arrow_schema = pa.Schema.from_pandas(pdf)
+
+        python_engine = python.Engine()
+
+        # Act
+        arrow_type = python_engine._convert_simple_pandas_dtype_to_offline_type(
+            arrow_schema.field("data").type
+        )
+        print(arrow_type)
+
+        # Assert
+        assert arrow_type == "int"
+
+    def test_infer_type_pandas_pyarrow_int32(self):
```

Comment:
it looks like these tests are duplicates, for example we already have this case covered: https://github.com/logicalclocks/feature-store-api/pull/1175/files#diff-01253ce273d2953406cf0ee612bf433ed4fdaae72f49126492cfe23841beb082L1147

Make sure there are no duplicates. And our naming for test methods is `test_[testedMethodName]_[condition]`. In this case it should be something like`test_convert_simple_pandas_dtype_to_offline_type_int32`